### ISSUE TITLE
energy media player for windows (win10)

### ISSUE THREAD

**Issue by terror1-3** [IDimitrovDev/Energy-Media-Player#4]
in mkv dv file multiple audio cant be selected please update

**Comment by IDimitrovDev** [comment]
Hey, unfortunately I don't seem to be able to simulate the selection issue. Do you have a sample file where the issue is presented?

**Comment by terror1-3** [comment]
here it is https://olamovies.rent/generate/?id=121299

**Comment by IDimitrovDev** [comment]
I've tried the test file and I was able to select the different audio streams getting different voice overs in that languages
![image](https://github.com/IDimitrovDev/Energy-Media-Player/assets/70667043/1806e555-7cdc-4894-9d4e-439945b5a608)
 What does it happen for you when you select the English audio stream for example?
 Also what are you options in the audio settings page for Dolby Digital
![image](https://github.com/IDimitrovDev/Energy-Media-Player/assets/70667043/d2005c19-778d-41f3-a00b-1ca136c54cd2)
and if passthrough is enable can you try disabling it and see if it then work?

**Comment by terror1-3** [comment]
if passthrough is disable i can see it works

**Comment by IDimitrovDev** [comment]
Passthrough leaves the audio decoding to the device that plays the audio - if its TV and tv output is selected in windows for playing from the TV speakers, but the TV doesn't support dolby digital plus with passthrough no audio will be played. When disabled the audio decoding is done by the player.

**Comment by terror1-3** [comment]
tv support atmos and vision , but why the video is also not playing

**Comment by IDimitrovDev** [comment]
Are you getting black screens when you try to play the video?
![image](https://github.com/IDimitrovDev/Energy-Media-Player/assets/70667043/dfe76092-d644-4a9d-ba59-54324841da3e)
Which option is selected here for you?

**Comment by terror1-3** [comment]
decode mode hw dolby vision
when selecting different audio apart from japanese which is has that forced flag the video looks like paused and not playing with hdr off. Again playing the same file from recent files it plays with only that japanese audio and not able to change audio again

**Comment by terror1-3** [comment]
yes i have tried that also ,while changing default audio to english first time it played and after that the same issue

**Comment by IDimitrovDev** [comment]
What is the CPU, RAM and GPU of the machine and how much ram/cpu is the player using while playing with english language and japanese. If its using big amount of memory you can try to disable prebuffering and enable disk caching which should improve memory usage
![image](https://github.com/IDimitrovDev/Energy-Media-Player/assets/70667043/fb65d184-5da5-4fb0-a462-c8c10d6e7ee0)
Also if you have multiple GPUs Windows by default selects the integrated one, but can be changed from the graphics settings
![image](https://github.com/IDimitrovDev/Energy-Media-Player/assets/70667043/894bf2c0-95d5-4ef5-af45-72211b7fca2b)
